airflow/configuration.py

```diff
@@ -468,12 +468,19 @@ class AirflowConfigParser(ConfigParser):
         ('celery', 'celery_result_backend')
     }
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, default_config=None, *args, **kwargs):
```

Comment:
`default_config` should come after `*args` so that it will be a keyword argument.